src/sentry/incidents/tasks.py

```diff
@@ -143,11 +156,13 @@ def handle_subscription_metrics_logger(subscription_update, subscription):
 
 
 @register_subscriber(INCIDENTS_SNUBA_SUBSCRIPTION_TYPE)
-def handle_snuba_query_update(subscription_update, subscription):
+def handle_snuba_query_update(
+    subscription_update: SubscriptionUpdate, subscription: QuerySubscription
+) -> None:
     """
     Handles a subscription update for a `QuerySubscription`.
     :param subscription_update: dict formatted according to schemas in
-    sentry_kafka_schemas.schema_types.events_subscription_results_v1.SubscriptionResults
+    sentry_kafka_schemas.schema_types.events_subscription_results_v1.SubscriptionResult
```

Comment:
This is making me question the `SubscriptionUpdate` typeddict I made in the previous PR, and I'm really confused but unfortunately don't have the time to keep working on this for now.

Reply:
You don't need these doc comments if there are types, should be a `SubscriptionUpdate` i think